CONTRIBUTING-development.md

```diff
@@ -0,0 +1,150 @@
+# Coding style
+
+* We almost always use **CamelCase**, when naming functions, variables etc., **not snake_case**.
+
+* We also have the habits of using a **lowerCamelCase** for some variables named from their type or from their function name, as shown in the examples:
+
+  ```c
+  AlterTableCmd *alterTableCommand = NULL;
```

Comment:
Global variables use UpperCamelCase, locals variables use lowerCamelCase.

Reply:
Ah, I now understand that this is some special case for a global where we don't, because the typename is so similar. Let's make it clear that this is a special case by showing an example of a normal global too.